src/simple_cache_client.rs

```diff
@@ -2714,6 +2714,19 @@ pub(crate) fn prep_request<R>(cache_name: &str, request: R) -> MomentoResult<ton
     Ok(request)
 }
 
+pub(crate) fn prep_request_with_timeout<R>(
```

Comment:
This obviously shouldn't be here, but I figured we could find a better home for this logic when we remove the old client.

Reply:
agreed